packages/cactus-plugin-ledger-connector-fabric/src/main/json/openapi.json

```diff
@@ -440,35 +459,108 @@
             "type": "string",
             "nullable": false
           },
-          "success": {
-            "type": "boolean",
+          "transactionId": {
+            "type": "string",
+            "nullable": false
+          }
+        }
+      },
+      "RunDelegatedSignTransactionRequest": {
+        "type": "object",
+        "required": [
+          "signerCertificate",
+          "signerMspID",
+          "channelName",
+          "contractName",
+          "invocationType",
+          "methodName",
+          "params"
+        ],
+        "additionalProperties": false,
+        "properties": {
+          "endorsingPeers": {
+            "description": "An array of endorsing peers (name or url) for the transaction.",
+            "type": "array",
+            "items": {
+              "type": "string",
+              "minLength": 1,
+              "maxLength": 4096,
+              "nullable": false
+            }
+          },
+          "endorsingOrgs": {
+            "description": "An array of endorsing organizations (by mspID or issuer org name on certificate) for the transaction.",
+            "type": "array",
+            "items": {
+              "type": "string",
+              "minLength": 1,
+              "maxLength": 4096,
+              "nullable": false
+            }
+          },
+          "transientData": {
+            "type": "object",
+            "nullable": true
+          },
+          "signerCertificate": {
+            "type": "string",
             "nullable": false
           },
-          "transactionId": {
+          "signerMspID": {
             "type": "string",
             "nullable": false
+          },
+          "uniqueTransactionData": {
+            "description": "Can be used to uniquely identify and authorize signing request",
+            "nullable": false
+          },
+          "channelName": {
+            "type": "string",
+            "minLength": 1,
+            "maxLength": 100,
+            "nullable": false
+          },
+          "contractName": {
+            "type": "string",
+            "minLength": 1,
+            "maxLength": 100,
+            "nullable": false
+          },
+          "invocationType": {
+            "$ref": "#/components/schemas/FabricContractInvocationType",
+            "nullable": false,
+            "description": "Indicates if it is a CALL or a SEND type of invocation where only SEND ends up creating an actual transaction on the ledger."
+          },
+          "methodName": {
+            "type": "string",
+            "minLength": 1,
+            "maxLength": 100,
+            "nullable": false
+          },
+          "params": {
+            "type": "array",
+            "nullable": false,
+            "default": [],
+            "items": {
+              "type": "string",
+              "nullable": true
+            }
+          },
+          "responseType": {
+            "$ref": "#/components/schemas/RunTransactionResponseType"
           }
```

Comment:
@outSH We **can** make breaking changes right now, but I would like to ask why do we have to remove these properties in the response object?
As-in, if we have good reason and cannot do any workaround to keep backward compatibility, then it is what it is, but otherwise I'm not so sure (even if we can make breaking changes, we should try to minimze the cognitive load for people who are migrating)

Reply:
@outSH Works for me, thank you very much taking the time to explain! It will be very helpful in the future!